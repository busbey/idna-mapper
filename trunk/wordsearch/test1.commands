swap column 0 5
