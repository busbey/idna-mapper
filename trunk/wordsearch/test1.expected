fbcdeaghijklmnop
fbcdeaghijklmnop
fbcdeaghijklmnop
fbcdeaghijklmnop
fbcdeaghijklmnop
fbcdeaghijklmnop
fbcdeaghijklmnop
